lib/redis/raceDetectionManager.js

```diff
@@ -43,8 +43,10 @@ export const onMutation = (collectionName, _id, fields, doc) => {
     // check for fields collision
     else if (fields && item.fields) {
       const common = _.intersection(item.fields, fields)
-      if (common.length && DeepDiff(_.pick(item.doc, common), _.pick(doc, common))) {
-        console.log('Redis-Oplog: Handling potential race condition on ' + uid + ': ' + common.join(', '))
```

Comment:
We do a diff after checking common.length as DeepDiff is expensive computationally